tests/console/yast2_ntpclient.pm

```diff
@@ -13,15 +13,18 @@
 use strict;
 use base "console_yasttest";
 use testapi;
+use utils qw(type_string_slow zypper_call);
 
 sub run {
     select_console 'root-console';
-
+    # Test often fails due to info ernel messages disrupting screen
+    # Decrease logging level to warning to avoid this
+    script_run "dmesg -n 4";
```

Comment:
that should not fail otherwise something is severly wrong, right? So I suggest just make it `assert_script_run`